gettingStartedWithAstraBlock/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+Stream real-time, human-readable blockchain data to your app with Astra Block.
+
+**In this guide, we'll**
+- Introduction to Astra Block
+- Example Queries
+- Request Access to Astra Block
+
+Let's get started!
+
+# Prerequisites
+None
+
+## 1 Introduction to Astra Block
+With Astra Block, you can stream real-time blockchain data directly to your Astra database. Simply enable the feature within Astra, create a blockchain database and start receiving data.
```

Comment:
I think the guide should include information about what the different tables are and why there are duplicate tables. Show how different tables are used depending on what info is available when making the query (transactions by transaction hash vs transactions by address vs transactions by block hash)